src/main/python/powerFactory2json/pf2json.py

```diff
@@ -0,0 +1,3153 @@
+            "iv_mode",
+            "kt",
+            "kt_a",
+            "ktc_a",
+            "limRampDown",
+            "limRampDownPU",
+            "limRampShutDown",
+            "limRampShutDownPU",
+            "limRampStartUp",
+            "limRampStartUpPU",
+            "limRampUp",
+            "limRampUpPU",
+            "loc_name",
+            "minDownTime",
+            "minUpTime",
+            "mode_inp",
+            "mode_pgi",
+            "mount",
+            "ngnum",
+            "nneutral",
+            "nphase",
+            "npnum",
+            "numBreakpointsCostOp",
+            "orient",
+            "orient_a",
+            "outServPzero",
+            "outserv",
+            "pBMU",
+            "pFlicker",
+            "pGRStoch",
+            "pOpCostCurve",
+            "pOperator",
+            "pOwner",
+            "pQlimType",
+            "pStoch",
+            "pStorage",
+            "p_over",
+            "p_under",
+            "pblocktrf",
+            "penaltyCosts",
+            "pf_over",
+            "pf_recap",
+            "pf_recap_a",
+            "pf_under",
+            "pfrecap",
+            "pgini",
+            "pgini_a",
+            "pginirnd",
+            "phiini",
+            "phmc",
+            "phtech",
+            "pini",
+            "pinput",
+            "pmaxratf",
+            "polyDegree",
+            "priority",
+            "psutype",
+            "qdslCtrl",
+            "qgini",
+            "qgini_a",
+            "qini",
+            "qinput",
+            "r0",
+            "r0hmc",
+            "r0iec",
+            "r1hmc",
+            "r2",
+            "r2hmc",
+            "r2iec",
+            "r2shc",
+            "root_id",
+            "rtox",
+            "sOpComment",
+            "scale0",
+            "scale0_a",
+            "scaleQmax",
+            "scaleQmin",
+            "searchBlockTrf",
+            "sernum",
+            "sgini_a",
+            "sgn",
+            "shfdiff",
+            "shfdiff_a",
+            "shfdir",
+            "shfdir_a",
+            "smoothfac",
+            "solalt_a",
+            "tds",
+            "tdss",
+            "tilt",
+            "tilt_a",
+            "timezone",
+            "typ_id",
+            "udeadblow",
+            "udeadbup",
+            "uk",
+            "umin",
+            "uonthr",
+            "usetp",
+            "usp_max",
+            "usp_min",
+            "vecBreakpointsP",
+            "vecCostRedispatchDown",
+            "vecCostRedispatchUp",
+            "vecPowerRedispatchDown",
+            "vecPowerRedispatchUp",
+            "vecStartUpCosts",
+            "vecStartUpTimes",
+            "x0",
+            "x0hmc",
+            "x0iec",
+            "x1hmc",
+            "x2",
+            "x2hmc",
+            "x2iec",
+            "x2shc",
+            "xtor"],
+}
+
+
+##########
+# Script #
+##########
+
+reserved_keywords = ["abstract",
```

Comment:
same here - having this in another file and just import/include it would be helpful